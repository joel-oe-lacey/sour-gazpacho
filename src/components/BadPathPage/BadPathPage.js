import React from 'react';
import './BadPathPage.scss';
import { Link } from 'react-router-dom';


const BadPathPage = () => {
    return (
      <main>
        <Link to="/"><button>Homepage</button></Link>
        <h1>PAGE NOT FOUND . . .</h1>
      </main>
    )
}

export default BadPathPage;
