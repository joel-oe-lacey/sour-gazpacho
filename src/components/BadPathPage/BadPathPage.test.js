import React from 'react';
import BadPathPage from './BadPathPage';
import { shallow } from 'enzyme'

describe('BadPathPage', () => {


  it('should match a snapshot as expected', () => {
    const wrapper = shallow(<BadPathPage />)

    expect(wrapper).toMatchSnapshot()
  })
})
