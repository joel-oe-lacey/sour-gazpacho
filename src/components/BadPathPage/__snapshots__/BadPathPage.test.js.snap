// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BadPathPage should match a snapshot as expected 1`] = `
<main>
  <Link
    to="/"
  >
    <button>
      Homepage
    </button>
  </Link>
  <h1>
    PAGE NOT FOUND . . .
  </h1>
</main>
`;
